JetStreamDriver.js

```diff
@@ -1531,6 +1531,32 @@ class WasmLegacyBenchmark extends Benchmark {
     }
 };
 
+function dotnetPreloads(type)
+{
+    return {
+        dotnetUrl: `./wasm/dotnet/build-${type}/wwwroot/_framework/dotnet.js`,
+        dotnetNativeUrl: `./wasm/dotnet/build-${type}/wwwroot/_framework/dotnet.native.js`,
+        dotnetRuntimeUrl: `./wasm/dotnet/build-${type}/wwwroot/_framework/dotnet.runtime.js`,
+        wasmBinaryUrl: `./wasm/dotnet/build-${type}/wwwroot/_framework/dotnet.native.wasm`,
```

Comment:
Some clarification questions on these generated Wasm binaries:
1. Several of them are bit-identical between the interpreter and AOT modes, e.g., `System.Text.Json.wasm` or `System.Drawing.Primitives.wasm`. Does that mean that core system/.NET libraries are always AOT-compiled, even in interpreter mode?
2. I don't see any `Program.wasm` file or similar for the AOT mode. To what is the benchmark compiled to? I assumed there would basically be one Wasm binary per CIL "binary" in AOT mode? (Forgive my ignorance, not a Blazor/.NET expert).

Reply:
Makes sense. Thanks for the explanation / background! :) Could you add this to the end of `README.md`, maybe under a section `## Background on Blazor / build output files`.